common/src/main/java/com/sequenceiq/cloudbreak/auth/ThreadBasedUserCrnProvider.java

```diff
@@ -1,12 +1,11 @@
-package com.sequenceiq.environment.configuration.security;
+package com.sequenceiq.cloudbreak.auth;
 
-import org.springframework.security.access.AccessDeniedException;
 import org.springframework.stereotype.Service;
 
 import com.sequenceiq.cloudbreak.auth.altus.Crn;
 
 @Service
-public class ThreadLocalUserCrnProvider {
+public class ThreadBasedUserCrnProvider {
```

Comment:
Github is stupid here. I didn't move the class. I removed the `ThreadLocalUserCrnProvider` from the Environment service and fixed the name of the `ThreadBaseUserCrnProvider`. Its correct name is: `ThreadBasedUserCrnProvider`. I also added the getAccountId convenience method to it.